tests/unit/test_core/test_interface.py

```diff
@@ -1,66 +1,69 @@
 """Interface tests."""
+# pylint: disable=attribute-defined-outside-init
 import logging
 import pickle
-import unittest
 from unittest.mock import MagicMock, Mock
 
+import pytest
 from pyof.v0x04.common.port import PortFeatures
 
 from kytos.core.common import EntityStatus
-from kytos.core.interface import TAG, UNI, Interface, TAGType
+from kytos.core.exceptions import (KytosSetTagRangeError,
+                                   KytosTagtypeNotSupported)
+from kytos.core.interface import TAG, UNI, Interface
 from kytos.core.switch import Switch
 
 logging.basicConfig(level=logging.CRITICAL)
 
 
-class TestTAG(unittest.TestCase):
+class TestTAG():
     """TAG tests."""
 
-    def setUp(self):
+    def setup_method(self):
```

Comment:
Appreciated you also taking the opportunity to fix the technical debt.

Reply:
No problem